CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 * Metric can be calculated on cutouts ([#149](https://github.com/KIT-IAI/pyWATTS/issues/149))
 
 ### Changed
+* Sampling with the Sampler-module is now possible in forwarding direction ([#174](https://github.com/KIT-IAI/pyWATTS/issues/174))
```

Comment:
I would assume that issue 174 is a bug. Therefore, please move this Changelog to the fixed section